skillingbeck/tfguard
# Various improvements

- Clarify usage in README (must run the terraform show command)
- Implement the allow-type-destroy flag
- Use the terms Allow and Block instead of Skip and Fail for more consistency
- Manual testing against terraform generated file
- Set up go.mod (since GitHub Actions $GOPATH does not appear to be properly set)
